SignalShareExtension/Info.plist

```diff
@@ -17,9 +17,9 @@
 	<key>CFBundlePackageType</key>
 	<string>XPC!</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>2.20.0</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>2.20.0.0</string>
```

Comment:
For now we're keeping this in lock-step with the main app manually?

Reply:
See https://github.com/WhisperSystems/Signal-iOS/pull/2831

I'm eventually also going to add a command line switch to change the release number.